args/core/ecs/entityquery.cpp

```diff
@@ -25,7 +25,7 @@ namespace args::core::ecs
 	{
 		hashed_sparse_set<id_type> componentTypes;
 		if (m_id)
-			componentTypes = m_registry.getComponentTypes(m_id);
+			componentTypes = m_registry.getComponentTypes(m_id); /// comment this glyn.
```

Comment:
I heard that talking to yourself is not healthy, I am unsure about writing yourself.